### ISSUE TITLE
[V3 Alias] Cannot run alias help

### ISSUE THREAD

**Issue by laggron42** [Cog-Creators/Red-DiscordBot#1792]
# Other bugs

#### What were you trying to do?

Run the `[p]alias help` command.

#### What were you expecting to happen?

Show the help for an alias, as the docstring says

#### What actually happened?

`[p]alias help x`
```Py
Exception in command 'alias help'
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/discord/ext/commands/core.py", line 62, in wrapped
    ret = yield from coro(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/redbot/cogs/alias/alias.py", line 292, in _help_alias
    is_alias, alias = self.is_alias(ctx.guild, alias_name=alias_name)
TypeError: 'coroutine' object is not iterable

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/discord/ext/commands/bot.py", line 886, in invoke
    yield from ctx.command.invoke(ctx)
  File "/usr/local/lib/python3.6/dist-packages/discord/ext/commands/core.py", line 927, in invoke
    yield from ctx.invoked_subcommand.invoke(ctx)
  File "/usr/local/lib/python3.6/dist-packages/discord/ext/commands/core.py", line 514, in invoke
    yield from injected(*ctx.args, **ctx.kwargs)
  File "/usr/local/lib/python3.6/dist-packages/discord/ext/commands/core.py", line 71, in wrapped
    raise CommandInvokeError(e) from e
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: TypeError: 'coroutine' object is not iterable
```

#### How can we reproduce this issue?

Try to run the command with any alias.

**Comment by Tobotimus** [comment]
Thanks for your bug report :smiley: This looks like a simple missing await.

**Comment by tekulvw** [comment]
Fixed with #1805
